TextAlarm/src/main/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+stripe.key.publishable = pk_test_uQ4QD76p0P5DGiNY6gegNiKv
```

Comment:
Your secret keys are now open to the internet.  Never check in production keys which your twilio keys count as.  Always use environment variable.